ndmg/stats/qa_fast.py

```diff
@@ -0,0 +1,207 @@
+import warnings
+
+warnings.simplefilter("ignore")
+import os
+import re
+import sys
+import numpy as np
+from numpy import *
+import nibabel as nb
+from scipy import ndimage
+from matplotlib.colors import LinearSegmentedColormap
+from nilearn.plotting.edge_detect import _edge_map as edge_map
+import matplotlib.pyplot as plt
+
+
+
+def opaque_colorscale(basemap, reference, vmin=None, vmax=None, alpha=1):
+    """
+    A function to return a colorscale, with opacities
+    dependent on reference intensities.
+    **Positional Arguments:**
+        - basemap:
+            - the colormap to use for this colorscale.
+        - reference:
+            - the reference matrix.
+    """
+    reference = reference
+    if vmin is not None:
+        reference[reference > vmax] = vmax
+    if vmax is not None:
+        reference[reference < vmin] = vmin
+    cmap = basemap(reference)
+    maxval = np.nanmax(reference)
+    # all values beteween 0 opacity and 1
+    opaque_scale = alpha * reference / float(maxval)
+    # remaps intensities
+    cmap[:, :, 3] = opaque_scale
+    return cmap
+
+def pad_im(image,max_dim,pad_val):
+
+    """
+    Pads an image to be same dimensions as given max_dim
+    Parameters
+    -----------
+    image: 3-d RGB np array of image slice
+    max_dim: dimension to pad up to
+    pad_val: value to pad with
+    Returns
+    -----------
+    padded_image: 3-d RGB np array of image slice with padding
+    """
+    pad_width = (((max_dim-image.shape[0])//2,(max_dim-image.shape[0])//2),((max_dim-image.shape[1])//2,(max_dim-image.shape[1])//2))
+    padded_image = np.pad(image, pad_width=pad_width, mode='constant', constant_values=pad_val)
+
+    return padded_image
+
+def plot_overlays(gm, csf, wm, cmaps=None, minthr=2, maxthr=95, edge=False):
+    '''
+    gm: 3d nparray of gm
+    csf: 3d nparray of csf
+    wm: 3d nparray of wm
```

Comment:
You need an actual docstring, formatted like the rest of the docstrings in `ndmg`. Please update this

Reply:
you did not do this, please don't mark things as resolved if they aren't resolved.